qiskit_optimization/applications/SK_model.py

```diff
@@ -0,0 +1,104 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+from typing import Optional, Union, List
+
+import networkx as nx
+import numpy as np
+from docplex.mp.model import Model
+
+from .graph_optimization_application import OptimizationApplication
+from ..algorithms import OptimizationResult
+from ..problems.quadratic_program import QuadraticProgram
+from ..translators import from_docplex_mp
+
+
+class SKModel(OptimizationApplication):
+    """Optimization application for the "Sherrington Kirkpatrick (SK) model" [1].
+
+    The SK Hamiltonian over n spins is given as:
+        H(x)=-1/\sqrt{n} \sum_{i<j} w_{i,j}x_ix_j, where x_i\in\pm 1
```

Comment:
Using the math formatting would be useful here to render the equation nicely.

Reply:
Can you please advice me what do you expect? Thank you.

Reply:
I think he is suggesting to use the math formatting like this example here does https://github.com/Qiskit/qiskit-optimization/blob/4347110991ba0ff656d20cbf50aaf1618ebd19b8/qiskit_optimization/problems/quadratic_program.py#L1043-L1049 which you can see how Sphinx formats it and its finally rendered here https://qiskit.org/documentation/optimization/stubs/qiskit_optimization.QuadraticProgram.from_ising.html

Reply:
Thank you. Done. Please check.

Reply:
Yes, this is what I had in mind. Looks good.